---
layout: post
title: Diving Into Undiagnosed Cases
subtitle: Thinking outside the box
tags: [Genomics, undiagnosed, rare diseases]
---

Rare genetic disorders are exeedingly difficult to diagnose, owing to the fact that over 7,000 of them exist (some estimates even higher around 10,000).  With exome sequencing, we can diagnose roughly 40% of cases (on average over multiple studies) where the germline mutation/inherited pathogenic variant is impacting the coding portion of the genome.  For the missing 60% of cases, what is going on?  Further more, what can we do?  

Turning to WGS is the obvious next step, but forging into the entire genome rapidly expands your search space for the "needle in the haystack".  You can look beyond the coding regions of the genome into the introns, 5'UTRs, 3'UTRs, ncRNAs, miRNAs, promoters, and cis regulatory elements. 

Even beyond that, we now have the ability to call larger and more complex variants that evaded detection from exome sequencing.  

As we forge into these new territories, several complications arise including the ability to distinguish artifacts from true genomic variants, and being able to assign impact to variants that aren't making predicted protein-coding changes.  This is the field in which my thesis lies, and with an ever-growing stack of unsolved/undiagnosed cases the pressure, is, on.

If you have interest in chatting about these types of things email me or tag me on twitter.  Both can be found at the bottom of the main page: https://phillip-a-richmond.github.io



